cluster/addons/fluentd-elasticsearch/es-statefulset.yaml

```diff
@@ -40,24 +89,22 @@ spec:
           name: transport
           protocol: TCP
         volumeMounts:
-        - name: elasticsearch-logging
+        - name: elasticsearch-data
```

Comment:
I thought `elasticsearch-logging` was good to associate the volume with the StatefulSet.